homeassistant/components/openai_conversation/const.py

```diff
@@ -41,6 +42,7 @@
 RECOMMENDED_WEB_SEARCH = False
 RECOMMENDED_WEB_SEARCH_CONTEXT_SIZE = "medium"
 RECOMMENDED_WEB_SEARCH_USER_LOCATION = False
+RECOMMENDED_WEB_SEARCH_INLINE_CITATIONS = True
```

Comment:
Let's make sure it's not the default. Default is people using voice with AI.
```suggestion
RECOMMENDED_WEB_SEARCH_INLINE_CITATIONS = False
```